### ISSUE TITLE
cifar10 train no gpu utilization, full gpu memory usage,  system cpu full loading

### ISSUE THREAD

**Issue by hmcmmmmm** [tensorflow/models#7339]
My test env:
1. Running command "python cifar10_train.py"  in ubuntu16.04
2. TF v1.13, 
3. gpu info as below 
![image](https://user-images.githubusercontent.com/24271691/62201521-2ce4ec80-b3ba-11e9-8aa9-a7b4c7ae83c6.png)
4. test log

-------------------------------------------------
Deprecated in favor of operator or tf.math.divide.
2019-07-31 17:26:38.155668: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2019-07-31 17:26:38.637076: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:998] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2019-07-31 17:26:38.637770: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x64e82a0 executing computations on platform CUDA. Devices:
2019-07-31 17:26:38.637784: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): Tesla V100-PCIE-16GB, Compute Capability 7.0
2019-07-31 17:26:38.655586: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 3696000000 Hz
2019-07-31 17:26:38.655760: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x60de020 executing computations on platform Host. Devices:
2019-07-31 17:26:38.655773: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): <undefined>, <undefined>
2019-07-31 17:26:38.656072: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1433] Found device 0 with properties:
name: Tesla V100-PCIE-16GB major: 7 minor: 0 memoryClockRate(GHz): 1.38
pciBusID: 0000:01:00.0
totalMemory: 15.75GiB freeMemory: 15.44GiB
2019-07-31 17:26:38.656085: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1512] Adding visible gpu devices: 0
2019-07-31 17:26:38.656831: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-07-31 17:26:38.656840: I tensorflow/core/common_runtime/gpu/gpu_device.cc:990]      0
2019-07-31 17:26:38.656844: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1003] 0:   N
2019-07-31 17:26:38.656929: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 15022 MB memory) -> physical GPU (device: 0, name: Tesla V100-PCIE-16GB, pci bus id: 0000:01:00.0, compute capability: 7.0)
2019-07-31 17:26:44.579378: I tensorflow/stream_executor/dso_loader.cc:152] successfully opened CUDA library libcublas.so.10.0 locally
2019-07-31 17:26:53.209515: step 0, loss = 4.68 (169.3 examples/sec; 1.512 sec/batch)
2019-07-31 17:27:05.408338: step 10, loss = 4.63 (209.9 examples/sec; 1.220 sec/batch)
2019-07-31 17:27:17.399277: step 20, loss = 4.43 (213.5 examples/sec; 1.199 sec/batch)
2019-07-31 17:27:29.413873: step 30, loss = 4.41 (213.1 examples/sec; 1.201 sec/batch)

**Comment by hmcmmmmm** [comment]
please give a suggestion, thanks

**Comment by tfboyd** [comment]
I would use [this version](https://github.com/tensorflow/models/tree/master/official/resnet) if on TensorFlow 1.x.  The current head needs tf-nightly but there are tags for each version of TensorFlow.  

If you are using TF nightly or TF 2.0 (nightly) you can use the keras version: https://github.com/tensorflow/models/tree/master/official/resnet/keras

For ResNet56 CIFAR-10 the keras version you should get ~3,400 images/sec FP32 on 1xV100

I test those versions nightly with many configurations and it would be easier to help you.
